server/dbSchemas/User.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
```

Comment:
I think we should delete this file since it's getting added by the other pr

Reply:
That makes sense,  @abearcher, could you remove this file in a follow up commit before we merge?